packages/components/src/components/progress-bar-onboarding/progress-bar-onboarding.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+
+import './progress-bar-onboarding.scss';
+
+type TProgressBarOnboarding = {
+    step: number;
+    amount_of_steps: Array<string>;
+    setStep: React.Dispatch<React.SetStateAction<number>>;
+};
+
+const ProgressBarOnboarding = ({ step, amount_of_steps, setStep }: TProgressBarOnboarding) => {
+    return (
+        <div className='progress-bar'>
+            {amount_of_steps.map((_, index) => {
+                const active = step === index + 1;
+
+                return (
+                    <div
+                        key={index}
+                        onClick={() => setStep(index + 1)}
+                        className={active ? 'progress-bar-rectangle' : 'progress-bar-circle'}
+                    />
+                );
+            })}
+        </div>
+    );
+};
+
+export default ProgressBarOnboarding;
```

Comment:
you need to export your component in `packages/components/src/index.js` so it can be usable across the app

Reply:
I am also exporting it using index.ts

Reply:
but we can not import components outside the Components package if we do not export it in the mentioned directory

Reply:
I've added our component to the index.js file